0002-data-collection-extension/README.md

```diff
@@ -104,19 +137,19 @@ Example:
 }
 ```
 
-If `id` field is present, the server MUST respond with a processing confirmation response.
+If the `id` field is present, the server MUST respond with a processing confirmation response.
 
-_Note: MQTT packet id can not be used. In that case we lose ability to work via gateways as a gateway is allowed to change MQTT packet id._
-
-In case batch does not specify id, the request degrades into a single-field JSON object.
+>**NOTE:** MQTT packet ID cannot be used.
+>If used, we would be unable to work via gateways as a gateway is allowed to change MQTT packet ID.
```

Comment:
It's best to avoid passive voice whenever possible. I rephrased it in active voice to avoid `we` but I wouldn't be too religious about it -- it's ok to sometimes use `we` in RFCs to indicate speaking on behalf of the community. See https://www.rfc-editor.org/ ("search RFCs").